### PR TITLE
Add unalias-env command

### LINKED ISSUE #284: Bosh CLI command that can remove an environment alias?
We are working on teardown scripts and `delete-env` does not remove the environment alias. Is there an argument that can be provided to `bosh alias-env` to remove a previously created alias? Or maybe there could be a new `delete-alias` command?

For now we are forced to manually clean up our `~/.bosh/config` but since we have multiple alias its an ugly shell script.

### PR DESCRIPTION
**Pull request by zmb3**
Fixes #284

### CODE REVIEW

**Comment by cppforlife** [comment]
@zmb3 should we name this command `unalias-env`?

**Comment by zmb3** [comment]
@cppforlife was actually going to do that, but Danny suggested delete-alias in the linked issue so I stuck with that. No strong opinions here.

**Comment by cppforlife** [comment]
will check in with danny tomorrow...

**Comment by dpb587-pivotal** [comment]
Grammatically/verbiage-wise... `delete-alias` makes more sense to me than `unalias-env`. But I do see the inconsistency between `alias-env` and `delete-alias`. Not sure there's a great choice, and I don't have a strong opinion either way.

**Comment by cppforlife** [comment]
@zmb3 all right, let's go with `unalias-env`.

**Comment by zmb3** [comment]
@cppforlife @dpb587-pivotal should be all set.

Despite the grammar I actually do like `unalias-env` - it sounds less destructive than `delete-alias`.

File: cmd/config/fs_config.go [hunk 1]
```diff
@@ -98,6 +98,26 @@ func (c FSConfig) AliasEnvironment(url, alias, caCert string) (Config, error) {
 	return config, nil
 }
 
+func (c FSConfig) DeleteAlias(alias string) (Config, error) {
```
Comment (cppforlife) [inline cmd/config/fs_config.go]: being picky... can we rename this to UnaliasEnvironment to be consistent with AliasEnvironment.

Comment (zmb3) [reply cmd/config/fs_config.go]: Sure.  I only left this one because from the perspective of the config we are actually _deleting_ the alias.

**Comment by zmb3** [comment]
friendly ping @cppforlife @dpb587-pivotal

**Comment by mfine30** [comment]
closing and re-opening so we can prioritize

**Pull request merged**